doc/posts/zitification/ssh/zitifying-ssh.md

```diff
@@ -0,0 +1,133 @@
+# Zitifying SSH
+
+As we learned in the [opening post](../README.md) - "zitifying" an application means to embed a Ziti SDK into an
+application and leverage the power of
+the [Ziti Network](https://openziti.github.io/ziti/overview.html#overview-of-a-ziti-network) to provide secure, truly
+zero-trust access to your application wherever in the world that application goes. In this post we are going to see how
+we have zitified `ssh` and why. Future posts will expand on this even further by showing how it is that NetFoundry
```

Comment:
Consider  changing `how it is that NetFoundry` to `how Netfoundry`

Reply:
👍